modules/ROOT/pages/ocis_release_notes.adoc

```diff
@@ -15,6 +15,136 @@
+* Enhancement - Configurable claims for auto-provisioning user accounts. We introduce the new environment variables "PROXY_AUTOPROVISION_CLAIM_USERNAME", "PROXY_AUTOPROVISION_CLAIM_EMAIL", and "PROXY_AUTOPROVISION_CLAIM_DISPLAYNAME" which can be used to configure the OIDC claims that should be used for auto-provisioning user accounts. The automatic fallback to use the 'email' claim value as the username when the 'preferred_username' claim is not set, has been removed. Also it is now possible to autoprovision users without an email address. #8635 #6909 #8952
+* Enhancement - Theme Processing and Logo Customization. We have made significant improvements to the theme processing in Infinite Scale. The changes include: Enhanced the way themes are composed. Now, the final theme is a combination of the built-in theme and the custom theme provided by the administrator via WEB_ASSET_THEMES_PATH and WEB_UI_THEME_PATH. - Introduced a new mechanism to load custom assets. This is particularly useful when a single asset, such as a logo, needs to be overwritten. - Fixed the logo customization option. Previously, small theme changes would copy the entire theme. Now, only the changed keys are considered, making the process more efficient. - Default themes are now part of ocis. This change simplifies the theme management process for web. These changes enhance the robustness of the theme handling in Infinite Scale and provide a better user experience. #8966 #9133
+* Enhancement - Add command to check ocis backup consistency. Adds a command that checks the consistency of an ocis backup. #9238
+* Enhancement - Web server compression. We've added a compression middleware to the web server to reduce the request size when delivering static files. This speeds up loading times in web clients. owncloud/web#7964 #9287
+* Enhancement - Activitylog Service. Adds a new service activitylog which stores events (activities) per resource. This data can be retrieved by clients to show item activities #9327
+
+
+[discrete]
+=== Migration
+
+- There is no upgrade path from previous releases, so this version is therefore only suitable for you if you start with a clean slate.
+
+[discrete]
+=== Breaking Changes
+
+- @mbarz please add
```

Comment:
Do before merging this? Or followup? Also, this is `@micbar` on github 😁  Unless you want to involve our dear friend @mbarz :heart: